Sources/Tropos/Views/DailyForecastView.swift

```diff
@@ -24,12 +24,4 @@ import TroposCore
         super.layoutSubviews()
         contentView.frame = bounds
     }
-
-    func setViewModel(dailyForecastViewModel: DailyForecastViewModel) {
```

Comment:
I think we still need this method! The Objective-C version of this, `-setViewModel:`, was a method that implemented the setter for the `viewModel` property. There are a few related features in Swift, but the way to have side effects on set in Swift is with a [property observer](https://docs.swift.org/swift-book/LanguageGuide/Properties.html#ID262).

That will look something like this:

```swift
@objc public var viewModel: DailyForecastViewModel? {
  didSet {
    dayOfWeekLabel.text = viewModel?.dayOfWeek
    //  . . .
  }
}
```

Reply:
Did not that was not objective c setters are defined... Implemented the suggested changes to the view model setter